src/abstracts/SablierV2Lockup.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity >=0.8.22;
 
-import { IERC4906 } from "@openzeppelin/contracts/interfaces/IERC4906.sol";
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IERC4906 } from "@openzeppelin/contracts/interfaces/IERC4906.sol";
```

Comment:
This should be above `IERC20` because path has higher priority over the imported contract name.

Reply:
https://github.com/sablier-labs/v2-core/pull/948/commits/65511d140a078b7f33cf2cd1308d74b4886d023c